File: psra/service/src/main/scala/by/arhor/psra/service/GalleryService.scala
package by.arhor.psra.service

import java.util

import by.arhor.psra.dto.GalleryDto
import by.arhor.psra.model.Gallery

trait GalleryService extends Service[Gallery, GalleryDto, String] {

  def findGalleriesByUserId(uid: String): util.List[GalleryDto]

}

File: psra/service/src/main/java/by/arhor/psra/service/GalleryService.java
package by.arhor.psra.service;

import by.arhor.psra.dto.GalleryDto;

import java.util.List;

public interface GalleryService
    extends Service<GalleryDto, String>
          , CanCreate<GalleryDto>
          , CanUpdate<GalleryDto> {

  List<GalleryDto> findGalleriesByUserId(String uid);

}
